src/zypper-plugins/enabled_repos_upload.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from os import environ
+import sys
+import logging, traceback
+
+from katello.repos import upload_enabled_repos_report
+from katello.enabled_report import EnabledReport
+from subscription_manager.repolib import ZypperRepoFile
```

Comment:
Are we guaranteed to have this available on older SLES like 11? I think we may need to fallback to a value we define if the ZypperRepoFile raises ImportError

Reply:
Good point have not installed these on SLES11